src/Agent.Plugins/Artifact/FileContainerProvider.cs

```diff
@@ -225,18 +231,32 @@ private async Task<IEnumerable<FileContainerItem>> GetArtifactItems(ArtifactDown
                 containerIdAndRoot.Item2
             );
 
-            IEnumerable<Func<string, bool>> minimatcherFuncs = MinimatchHelper.GetMinimatchFuncs(
-                minimatchPatterns,
-                tracer,
-                downloadParameters.CustomMinimatchOptions
-            );
+            Options customMinimatchOptions;
+            if (!(downloadParameters.CustomMinimatchOptions is null))
```

Comment:
will be better to use this form `if (downloadParameters.CustomMinimatchOptions != null)`

Reply:
Corrected